### DISCUSSION TITLE
Adjusting Memory Parameters

### DISCUSSION THREAD

**Discussion by ethanic17** [itzg/docker-minecraft-server#1287] [Q&A] [answered]
I'm planning to lower the memory allocation of the mc server/container from 2GB to 1GB (yes, I know it isn't recommended), and since I'm editing the env variables, I would have to recreate and deploy the container again. This wouldn't mess up anything right? Like I'll still do a backup of all the files but once I start the new container, all the files in the mc server and my world will be intact correct?

**Comment by itzg** [comment, marked as answer]
If you attached `/data` to a host directory or persistent volume then yes, everything remains intact.

**Reply by ethanic17** [reply]
I believe I did so, but just to double check:
![image](https://user-images.githubusercontent.com/41976386/149710153-d04798c6-b76a-4744-9a39-5fc01c2ae588.png)

**Reply by itzg** [reply]
BTW you didn't need to use literally `/path/on/host`. That was a fake example path, but you can use whatever path you want that exists in your host.

**Reply by ethanic17** [reply]
ah sweet, thank you.
